bulbs/special_coverage/models.py

```diff
@@ -83,10 +85,8 @@ def save(self, *args, **kwargs):
         super(SpecialCoverage, self).save(*args, **kwargs)
 
         if self.query and self.query != {}:
-            if self.is_active:
-                self._save_percolator()
-            else:
-                self._delete_percolator()
+            # Always save and require client to filter active date range
+            self._save_percolator()
```

Comment:
This fixes a regression w/ new is_active logic where a SpecialCoverage start_time would become active but it wouldn't be in percolator (b/c when it was last saved it wasn't active)